ext/uri/uriparser/src/UriCommon.h

```diff
@@ -70,7 +70,6 @@
 
 /* Used to point to from empty path segments.
  * X.first and X.afterLast must be the same non-NULL value then. */
-extern const URI_CHAR * const URI_FUNC(SafeToPointTo);
```

Comment:
Why was this removed/moved?

Reply:
This is the background story: https://github.com/php/php-src/pull/18836#discussion_r2140807830